### PR TITLE
airframes add new optional include tag

### PR DESCRIPTION
**Pull request by dagar**

### CODE REVIEW

**Comment by dagar** [comment]
Later the include could be expanded to apply across all airframes. Boards like the crazyflie, and intel aero shouldn't have to include anything other than the specific airframes.

Capabilities is another interesting one to think about for the number of possible outputs. The pixracer shouldn't show any options that require more outputs (or AUX outputs) than it physically has.

**Comment by hamishwillee** [comment]
We need to document what this is for. Possibly add it to http://dev.px4.io/en/airframes/adding_a_new_frame.html (or 

We should as part of this doc the metadata supported. So far I see:

Param | Example | Description
--- | --- | ---
`@name` | Wing Wing | Unique name of vehicle frame (product name)
`@url` | https://docs.px4.io/en/framebuild_plane/wing_wing_z84.html |  Link to build log
`@class` | Copter, Plane, Rover, VTOL |  Broad airfame type - see airframe reference
`@type` | Octorotor Coaxial, Octorotor x, Simulation, Flying Wing, Standard Plane, ...  | Airframe sub-type - see airframe reference
`board` | `@board px4fmu-v3 include` |  ?
`@maintainer` | `@maintainer Andreas Antener <andreas@uaventure.com>` |  email address of maintainer
`@output` | @output MAIN1 motor 1 |  Document mapping of FC board outputs (MAIN, AUX) to a specific motor.

Questions:
- Above look OK?
- What is mandatory/optional to specify
- What is `@board` for? 
- What does `@board` do without `include`
- Are there any other fields?

**Comment by LorenzMeier** [comment]
Rebased.

File: ROMFS/px4fmu_common/init.d/4030_3dr_solo [hunk 1]
```diff
@@ -5,11 +5,7 @@
 # @type Quadrotor x
 # @class Copter
 #
-# @board px4fmu-v2 exclude
-# @board px4fmu-v4 exclude
-# @board px4fmu-v4pro exclude
-# @board px4fmu-v5 exclude
-# @board aerofc-v1 exclude
+# @board px4fmu-v3 include
```
Comment (LorenzMeier) [inline ROMFS/px4fmu_common/init.d/4030_3dr_solo]: @dagar Flash is overflowing now with this change - which is a hint to me that it is not working as designed.

**Pull request closed without merging**